### PR TITLE
sls-app bug fixes

### PR DESCRIPTION
**Pull request by swati-sh**
Description of the Change

* Removed the asset name , when there is no asset identifier is available.

### CODE REVIEW

File: core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts [hunk 1]
```diff
@@ -470,7 +471,18 @@ export class ServiceLogsComponent implements OnInit {
 		$(".pagination.justify-content-center li:nth-child(2)")[0].click();
 		this.callLogsFunc();
 	}
-
+	assetsModify(){
```
Comment (raghits) [inline core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts]: Can we name this method to be more indicative of the purpose?

File: core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts [hunk 2]
```diff
@@ -470,7 +471,18 @@ export class ServiceLogsComponent implements OnInit {
...
+		var value = (<HTMLInputElement>document.getElementById('allasset'))
+		var resValue = (<HTMLInputElement>document.getElementById('Allidentifier'))
+		if(value != null) {
+			var inputValue = value.checked = true;
```
Comment (raghits) [inline core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts]: do we need to assign to inputValue?

File: core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts [hunk 3]
```diff
@@ -470,7 +471,18 @@ export class ServiceLogsComponent implements OnInit {
...
+		}
+		if(resValue != null) {
+			var input = resValue.checked = true;
```
Comment (raghits) [inline core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts]: do we need to assign to input?

Comment (swati-sh) [reply core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts]: No,done the changes

File: core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts (hunk 2 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts]: done the changes

File: core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts (hunk 1 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/pages/service-logs/service-logs.component.ts]: done

**Pull request merged**